ui/components/multichain/pages/permissions-page/connection-list-item.js

```diff
@@ -25,14 +26,22 @@ import {
 } from '../../../component-library';
 import { getURLHost } from '../../../../helpers/utils/util';
 import { SnapIcon } from '../../../app/snaps/snap-icon';
-import { getPermittedChainsForSelectedTab } from '../../../../selectors';
+import { getAllPermittedChainsForSelectedTab } from '../../../../selectors';
 
 export const ConnectionListItem = ({ connection, onClick }) => {
   const t = useI18nContext();
   const isSnap = connection.subjectType === SubjectType.Snap;
-  const connectedNetworks = useSelector((state) =>
-    getPermittedChainsForSelectedTab(state, connection.origin),
+  const _permittedChains = useSelector((state) =>
```

Comment:
How does `permittedChains` differ from `_permittedChains`?

Reply:
I've updated the selector to handle the filtering of the `wallet` scopes and added a comment explaining: https://github.com/MetaMask/metamask-extension/pull/29887/commits/99f44c2eb47584e1415b2592dcd390ef020ecea9